### DISCUSSION TITLE
为什么获取mod要等待2分钟以上

### DISCUSSION THREAD

**Discussion by bukanxingxingma** [Meloong-Git/PCL#2544] [提问] [answered]
不管是搜索还是刷新mod列表 基本要2-5分钟才能展现

**Comment by SuoHouDaiShi** [comment]
一般是网络不佳 请在PCL的设置中改为尽量使用镜像源就行

Edit:也可以使用VPN或者加速器...

**Comment by WForst-Breeze** [comment]
如果是 mod 管理刷新比较慢，感觉可能是设备的原因🤔

如果是 mod 下载获取比较慢，应该是网络环境问题，可以把来源调整成 modrinth，可能会好一点

**Comment by bukanxingxingma** [comment, marked as answer]
我发现是DNS的祸




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月21日(星期一) 上午9:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Hex-Dragon/PCL2] 为什么获取mod要等待2分钟以上 (Discussion #2544)





 
一般是网络不佳 请在PCL的设置中改为尽量使用镜像源就行

**Reply by SuoHouDaiShi** [reply]
那你close掉 谢谢
